neoscore/interface/image_interface.py

```diff
@@ -36,6 +38,9 @@ class ImageInterface(PositionedObjectInterface):
     z_index: int = 0
     """Z-index controlling draw order."""
 
+    transform_origin: Point = ORIGIN
+    """Axis of rotation"""
```

Comment:
This isn't just the rotation axis, it's the axis for all transformations. For our purposes, that means scaling and rotation. A better docstring would be something like "The origin point for rotation and scaling transforms"